VK/preregistration.qmd

```diff
@@ -0,0 +1,338 @@
+        - Separate measurement model per latent variable (b = k = 5)
+        - Joint measurement model for all exogenous variables (b = 3)
+
+**Explanation of the chosen factor values:**
+
+The aim is a conceptual replication of the results obtained by @dhaene_evaluation_2023 in study 1 and @rosseel_structural_2022 study 2. The values were chosen accordingly. No detailed explanation was provided by the authors.
+
+### If there is more than one factor: How will the factor levels be combined and how many simulation conditions will this create?
+
+**Answer:**
+- Study 1:
+    - Fully factorial design with 4 models x 3 sample sizes x 3 reliabilities = 36 conditions
+- Study 2.1:
+    - Fully factorial design with 2 models x 3 sample sizes x 3 reliabilities x 2 distributions = 36 conditions
+- Study 2.2:    
+    - Fully factorial design with 2 models x 3 sample sizes x 5 misspecifications = 30 conditions
+
+## 4. Estimands and Targets
+
+### Estimands
+
+Structural model parameters (path coefficients)
+
+## 5. Methods
+
+Both studies will compare four different estimation methods for SEMs:
+
+- **"Vanilla" SEM**: (structural and measurement model estimated simultaneously) (rationale: the current standard approach in SEM estimation serving as a baseline with maximum likelihood (ML)):
+- **SAM**: (separating the estimation of the measurement and structural model to alleviate the potential for propagation of bias from (e.g. misspecified) measurement part to the structural part of the model)
+    - **Local SAM** (Uses summary statistics from the measurement model to derive the model-implied mean vector and variance-covariance matrix of latent variables. These statistics are then utilized to estimate the structural parameters. A mapping matrix (M) is used to transform the observed data into the latent variable space. It can be estimated using different methods.)
+        - **With ML mapping matrix** (Akin to a factor score approach (@bartlett_statistical_1937, @bartlett_methods_1938))
+        - **With ULS mapping matrix** (Uses the Moore-Penrose pseudoinverse, suitable for scenarios with complex or underdetermined systems, where the K matrix is rank-deficient but requires adjustments for structural constraints.)
+    - **Global SAM** (rationale: Fixing the parameters obtained from the measurement model in the first step, and then using them as constants in the full SEM during the second step. Suitable for models where local SAM is impractical due to higher-order latent variables or rank deficiencies in $\lambda$.)
+
+"Vanilla" SEM as well as both steps in the SAM approach will be estimated using Maximum Likelihood (ML) using the `lavaan` (@rosseel_lavaan_2012) package in R (@r_core_team_r_2023).
+
+## 6. Performance Measures
+
+Across both studies the following performance measures will be captured:
+
+- Convergence rates: Proportions of observed data sets that successfully converged for each estimation method.
```

Comment:
Maybe you should also state what you do with non-converged cases for calculating bias and RMSE? Repeat, impute?